sass/tabzilla.scss

```diff
@@ -0,0 +1,39 @@
+@import 'bourbon';
```

Comment:
\*shrug\* It does seem like overkill. We could just make a custom mixin to do the media query (or hard code it).